kratos/geometries/geometry.h

```diff
@@ -2027,6 +2055,22 @@ class Geometry
         return rResult;
     }
 
+    /**
+     * @brief This method provides the global coordinates corresponding to the given integration method.
+     * @param rResult The array containing the global coordinates corresponding to the given integration method.
+     * @param IntegrationPointIndex index of integration point which jacobians has to be calculated in it.
+     * @param rIntegrationPointsArray The integration method which jacobians has to be calculated in its integration points.
+     * @return An array containing the global coordinates corresponding to the given integration method.
+     */
+    virtual CoordinatesArrayType& GlobalCoordinates(
```

Comment:
why virtual? Is this not general enough when the main function GlobalCoordinates is already virtual?